gymnasium/envs/mujoco/mujoco_env.py

```diff
@@ -309,7 +312,7 @@ def render(self):
 
     def _get_viewer(
         self, mode
-    ) -> Union["mujoco_py.MjViewer", "mujoco_py.MjRenderContextOffscreen"]:
+    ) -> mujoco_py.MjViewer | mujoco_py.MjRenderContextOffscreen:
```

Comment:
This requires `mujoco_py` to be imported,
if `mujoco_py` is not imported does this cause issues?